lisp/core.slosh

```diff
@@ -807,6 +807,40 @@ Example:
 %#
 (defn identity (x) x)
 
+#%
+Usage: (not= arg1 arg2)
+
+Test if two values are not equal using `=`
+
+Section: core
+
+Example:
+(assert-true (not= 0 1))
+(assert-true (not= 1 1.0))
+(assert-false (not= 2 2))
+(assert-false (not= 0.0 -0.0))
+%#
+(defmacro not= (expected-val right-val & body)
```

Comment:
Get rid of "& body" it is not used.  This is adding a final parameter that will collect any extra params on a call, that is not what you want to do.  That would allow something like (not= 1 2 "blah" 4 8 6) to compile and be equivalent to (not= 1 2).  Macros often use this when they are wrapping an undetermined list of code.

Reply:
thanks for the explanation!